src/elf/mod.rs

```diff
@@ -202,22 +202,45 @@ if_sylvan! {
         pub fn is_object_file(&self) -> bool {
             self.header.e_type == header::ET_REL
         }
+
+        pub fn parse_hdr(bytes: &'a [u8]) -> error::Result<Header> {
+            bytes.pread::<Header>(0)
+        }
+
+        // Lazy parse the ELF contents. This function mainly just assembles an Elf struct. Once we have the struct, we can choose to parse whatever we want.
```

Comment:
this is also missing an extra /, just noticed it

Reply:
Done.